src/doc/src/reference/environment-variables.md

```diff
@@ -23,6 +23,12 @@ system:
 * `CARGO_TARGET_DIR` — Location of where to place all generated artifacts,
   relative to the current working directory. See [`build.target-dir`] to set
   via config.
+* `CARGO` - If set, Cargo will forward this value instead of setting it
+  to its own auto-detected path when it builds crates and when it
+  executes build scripts and external subcommands. This value is not
```

Comment:
Should we include that the value, if set, should be the path to the `cargo` binary performing the build?

Reply:
I don't think that's necessary. It may also be confusing phrasing for tools that use Cargo as a library but do not otherwise have a `cargo` binary to point to (e.g., because they're not `cargo` subcommands).